package.json

```diff
@@ -7,29 +7,36 @@
   "scripts": {
     "prepublish": "require-npm4-to-publish",
     "prepublishOnly": "npm test",
-    "pretest": "tsc",
+    "pretest": "npm run build",
+    "build": "npm run prettify:src && tsc && npm run prettify:dist",
+    "prettify:src": "prettier --use-tabs --single-quote --write src/*.ts",
+    "prettify:dist": "prettier --use-tabs --single-quote --write src/*.js",
     "test": "mocha --compilers ts:ts-node/register && mocha --compilers coffee:coffee-script/register"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/resin-io/typed-error.git"
   },
   "devDependencies": {
-    "@types/bluebird": "^3.0.36",
-    "@types/mocha": "^2.2.33",
-    "@types/node": "^6.0.48",
-    "bluebird": "^3.4.6",
-    "chai": "~1.9.0",
-    "chai-as-promised": "~4.1.1",
-    "coffee-script": "~1.12.0",
-    "mocha": "~1.21.3",
+    "@types/bluebird": "^3.5.8",
+    "@types/mocha": "^2.2.41",
+    "@types/node": "^8.0.0",
+    "bluebird": "^3.5.0",
+    "chai": "^4.1.0",
+    "chai-as-promised": "^7.1.1",
+    "coffee-script": "~1.12.7",
+    "mocha": "^3.4.0",
+    "prettier": "^1.5.3",
     "require-npm4-to-publish": "^1.0.0",
-    "ts-node": "^1.7.2",
-    "typescript": "^2.1.4"
+    "ts-node": "^3.2.0",
+    "typescript": "2.4.2"
```

Comment:
Yes, we've had an issue recently in the UI when the patch bump has broken the builds